### PR TITLE
[LC-482] change 'get status' communication method (gRPC -> rabbitMQ)

### PR DESCRIPTION
**Pull request by metalg0su**
# Goal
- APIs `Status`, `Avail` now get status of loopchain by using Earlgrey MQ connection, not using gRPC stub call.
- Remove dependencies

(This PR does not require changes in Loopchain)

### CODE REVIEW

File: iconrpcserver/server/rest_server.py [hunk 1]
```diff
@@ -128,19 +127,28 @@ def serve(self, api_port):
         self.__app.run(host='0.0.0.0', port=api_port, debug=False, ssl=self.ssl_context)
 
 
+def get_status_by_earlgrey(channel_name) -> dict:
+    channel_stub = StubCollection().channel_stubs[channel_name]
+    status_data: dict = channel_stub.sync_task().get_status()
```
Comment (zsaladin) [inline iconrpcserver/server/rest_server.py]: async task is better

Comment (windies21) [reply iconrpcserver/server/rest_server.py]: How about renaming 'get_status_by_earlgrey' to 'get_channel_status' or something else?

**Review by windies21 — APPROVED** [review]

File: iconrpcserver/server/rest_server.py (hunk 1 above)
Comment (metalg0su) [reply iconrpcserver/server/rest_server.py]: @zsaladin 
> async task is better

Thank you! I applied async and await keywords.

@windies21 
> How about renaming 'get_status_by_earlgrey' to 'get_channel_status' or something else?

Reasonable name. Thank you!

**Review by yakkle — APPROVED** [review]
Please squash commit if possible.

**Pull request merged**